src/docfx/build/toc/BuildTableOfContents.cs

```diff
@@ -24,11 +24,6 @@ internal static class BuildTableOfContents
             var dependencyMapBuilder = new DependencyMapBuilder();
             var (errors, tocModel, tocMetadata, refArticles, refTocs) = Load(context, file, dependencyMapBuilder);
```

Comment:
what if a link in TOC points to an PNG or config? we could have a TODO here.

Reply:
toc can't reference to a link or config right now, but i can add a TODO here.

---
In reply to: [227639083](https://github.com/dotnet/docfx/pull/3519#discussion_r227639083) [](ancestors = 227639083)